src/index.html

```diff
@@ -30,6 +30,16 @@ <h1 id="logo-text" class="text-center">
         </div>
       </header>
       <section>
+        <form name="myForm" class="form">
+          <div class="row justify-content-center">
+            <div class="justify-content-center">
+              <input type="radio" id="radio-ethers" name="libraryUsed" value="ethers" />
+              <label id="libraryLabel">Ethers</label>
```

Comment:
Replace `id` with `class` here for both labels.

Reply:
Fixed in: 91555c04c8088ee472c87cc25843642994ec92b4